openseespy-pip/openseespy/postprocessing/Get_Rendering.py

```diff
@@ -582,8 +585,7 @@ def nodecoordsEigen(nodetag):
 	return fig, ax
 	
 
-
-def plot_deformedshape(Model="none", LoadCase="none", tstep = -1, scale = 10, overlap='no', monitorEleTags = [], monitorOutFile = "none"):
```

Comment:
I guess you need to update your fork to match this branch. We'll keep monitorEleTags input for now. It has been helpful for big structures for my research. I think this could help others.

Reply:
Yeah, this should be there.. Let me confirm. I might need to remove this pull request.

Reply:
This should be fixed.